src/Controller.php

```diff
@@ -34,15 +48,45 @@ public function sync(OutputInterface $output, GlobalConfig $config, SyncOptions
             return;
         }
 
+        $sectionTop = null;
+        $sectionBottom = null;
+
         if ($output instanceof ConsoleOutputInterface) {
-            $sectionTop = $output->section();
-            $sectionMiddle = $output->section();
-            $sectionMiddle->writeln(str_repeat('-', 20), OutputInterface::OUTPUT_RAW);
-            $sectionBottom = $output->section();
-        } else {
-            $sectionTop = $output;
```

Comment:
this functionality should be possible afterall.
``\PHPSu\Controller`` is API for other php code and should allow all kinds of ``OutputInterface``